camel/types/enums.py

```diff
@@ -184,6 +184,7 @@ def output_dim(self) -> int:
 class TaskType(Enum):
     AI_SOCIETY = "ai_society"
     CODE = "code"
+    GITHUB = "github"
```

Comment:
will github be a high-level Task for one specific agent to solve rather than a tool/function can be called by agent?

Reply:
GitHub Agent will perform a range of different tasks related to Github. In this PR it's retrieving the last issue and solving it, the next task will be getting the list of merged PRs and providing a summary that later can be posted on social media.